### ISSUE TITLE
'Other' greenhousegas emissions

### ISSUE THREAD

**Issue by Charlottevm** [quintel/etmodel#4138]
Currently the 'Emissions' side bar item is somewhat confusing as users think they can change actual energy emissions here.
Furthermore, other emission types are scattered throughout the ETM (such as emissions on electricity import or heat). 

Proposal:

Rename 'Greenhouse gases' into 'Other greenhouse gases' & place all other emission sliders under this sidebar item (e.g. by creating a new sidebar item 'Emission factors').

What do you think @redekok @mabijkerk ?

**Comment by mabijkerk** [comment]
I'm in favour of clarifying this, as it is currently confusing for our users. However, I think discussion is needed what the best solution is.

As for the Emission factors, this may be something that can be picked up within the Transit flows project, as it will restructure import flows.

**Comment by Charlottevm** [comment]
Sounds good, shall we have a (live) discussion about this during that project @mabijkerk ?

**Comment by redekok** [comment]
I agree with both of you! This definitely needs clarification. Discussing this during the transit flows project sounds like a great idea.
